imports/ui/JapaneseNewGame.js

```diff
@@ -720,42 +720,44 @@ function push_dealin_hand(template) {
 	var riichiSum = Session.get("free_riichi_sticks");
 	var eastDelta = 0, southDelta = 0, westDelta = 0, northDelta = 0;
 
-	if 		(winnerWind == "east") {
+	if (winnerWind == Constants.EAST) {
 		Session.set("eastPlayerWins", Number(Session.get("eastPlayerWins")) + 1);
 		Session.set("eastPlayerPointsWon", Number(Session.get("eastPlayerPointsWon")) + points);
 		Session.set("eastPlayerDoraSum", Number(Session.get("eastPlayerDoraSum")) + dora);
-		if (Session.get("east_riichi") == true) 
+		if (Session.get("east_riichi") == true) {
 			Session.set("eastPlayerRiichisWon", Number(Session.get("eastPlayerRiichisWon")) + 1);
-	}
-	else if (winnerWind == "south") {
+		}
+	} else if (winnerWind == Constants.SOUTH) {
```

Comment:
Force of habit, although keep in mind that all this change does is modify it so that the oneline if statement is encapsulated in curly braces instead of following in a logical line.